Library/ViewModels/DiscoveryFiltersViewModel.swift

```diff
@@ -140,6 +143,10 @@ public final class DiscoveryFiltersViewModel: DiscoveryFiltersViewModelType,
       )
       .map { (rows: $1, categoryId: $0.0, selectedRowId: $0.1) }
 
+    self.loadCategoryRows.signal.observeValues { values in
+      print(values)
```

Comment:
You probably don't need this `print` anymore.

Reply:
good catch. this line should have been deleted.